### ISSUE TITLE
Question about the Training Time

### ISSUE THREAD

**Issue by ameya98** [atomistic-machine-learning/G-SchNet#8]
Hi, we were wondering if you noted how long (number of iterations or epochs) you trained your model for? What GPU did you train on? How long (in days) did training take? It would be very helpful for reproducibility!

**Comment by NiklasGebauer** [comment]
Hi Ameya,
sorry for the late answer, I had to dig a bit in the archives.
Training stopped after 323 epochs as the learning rate reached 1e-6 (we used early stopping to select the model with the lowest validation loss, therefore the employed model was probably trained for around 300 epochs).
I cannot exactly say which GPU was used but it was either an nvidia P100 or V100 with 16gb of VRAM and it took 4 days and 5 hours (a little less than 20 minutes per epoch).
Hope this helps!
